x/xion/keeper/msg_server.go

```diff
@@ -124,7 +125,7 @@ func (k msgServer) MultiSend(goCtx context.Context, msg *types.MsgMultiSend) (*t
 		outputs = append(outputs, banktypes.NewOutput(feeCollectorAcc, totalPlatformCoins))
 	}
 
-	err := k.bankKeeper.InputOutputCoins(ctx, msg.Inputs, outputs)
+	err := k.bankKeeper.InputOutputCoins(ctx, msg.Inputs[0], outputs) // TODO decide on how to handle multiple inputs
```

Comment:
What does the existing Bank MultiSend do in this instance? This was basically a straight copy

Reply:
the logic stayed the same, in the cosmos sdk we opted to remove the validatebasic function and instead do the checks in the msg execution phase. the approach that is done here is still safe and the logic stays the same.